monthorders.cpp

```diff
@@ -266,6 +266,12 @@ Location *Game::Do1SailOrder(ARegion *reg, Object *fleet, Unit *cap)
 			newreg = reg->neighbors[x->dir];
 		}
 		cost = 1;
+
+		// Blizzard effect
+		if (newreg && newreg->weather == W_BLIZZARD && !newreg->clearskies) {
```

Comment:
this is a part of `Do1SailOrder` orders

Reply:
Why do you think it is necessary to limit ships so much? What are the considerations?

Reply:
The idea of freezing is to narrow down a map towards the central belt. Freezing oceans and lands is consistent and MP is 4MP for both, so horse and ship will move with the same speed